cfpb/consumerfinance.gov
# Hide "CDN Tools" link in Wagtail if not enabled

Currently the "CDN Tools" link appears in the Wagtail sidebar whether or not CDN purging is configured. If it is not enabled, we show a separate template that says

> CDN management is not currently enabled on this server. Please see your administrator if you have any questions.

This commit removes this view in favor of hiding the link if the CDN is not enabled.

## How to test this PR


To test this change locally, to simulate running with a configured CDN, run this command:

```
ENABLE_AKAMAI_CACHE_PURGE=1 ./runserver.sh
```

Then visit http://localhost:8000/admin/cdn/ and confirm you have a "CDN Tools" link in the Wagtail sidebar and the CDN view can be loaded.

If instead you just run:

```
./runserver.sh
```

you should have no such link and http://localhost:8000/admin/cdn/ should return a 404.

## Screenshots

|Before|After|
|-|-|
|<img width="1552" alt="image" src="https://user-images.githubusercontent.com/654645/232609529-d41f32e7-b1ed-44e2-8150-b89b552470b7.png">|<img width="1552" alt="image" src="https://user-images.githubusercontent.com/654645/232609947-628751d0-f609-4f92-933e-e2671516d498.png">|

## Checklist
